infrastructure/main.tf

```diff
@@ -6,7 +6,7 @@ locals {
 
 
 module "div-validation-service" {
-  source                          = "git@github.com:hmcts/moj-module-webapp.git"
+  source                          = "git@github.com:hmcts/cnp-module-webapp.git"
```

Comment:
as per @hosainnet 's comment in different PR, we might need to follow the format done in the following PR
[hmcts/div-case-orchestration-service@5c2f6ed](https://github.com/hmcts/div-case-orchestration-service/commit/5c2f6ed90b78b74ad7a29f7e07f2f3627ec1312c)